src/js/components/products/listItem.tsx

```diff
@@ -31,25 +31,25 @@ const ProductItem = ({ item }: { item: Product }) => {
           icon={<ProductIcon item={item} />}
           bodyClassName="slds-card__body_inner"
         >
-          <div>
-            <span className="slds-text-title slds-m-right_xx-small">
+          <div className="md-product-meta slds-grid slds-wrap slds-grid_vertical-align-center">
+            <span className="slds-text-title slds-m-right_small slds-p-vertical_xx-small md-product-meta-title">
               {t('Version {{version}}', { version: label })}
             </span>
             {window.GLOBALS.SITE?.show_product_tags &&
               item.tags.map((tag, idx) => (
                 <span key={idx}>
                   <Pill
                     className="slds-p-around_xx-small
-                      slds-m-vertical_xx-small
-                      slds-m-right_xx-small"
+                      slds-m-right_small
+                      md-pill-plain"
```

Comment:
This looks unused?

Reply:
@jgerigmeyer Thanks for the reminder. I wrote the CSS first before I added this class to the markup and forgot to go back and update the CSS. It's fixed now.